.env.example

```diff
@@ -115,3 +121,23 @@ AIRFLOW_SERVICE_SCHEME=http
 AIRFLOW_SERVICE_HOST=airflow-webserver
 AIRFLOW_SERVICE_PORT=8080
 AIRFLOW_SERVICE_PATH_PREFIX=/api/v1
+
+# Search Service
+ES_HOST=badgerdoc-elasticsearch
+ES_PORT=9200
+APP_TITLE=Badgerdoc Search
```

Comment:
Do we need to use `APP_TITLE` in search microservice? Or it's badgerdoc configuration?